_rules/image-no-text-0va7u6.md

```diff
@@ -33,15 +33,17 @@ This rule applies to any [visible][] [embedded image][] that is not [marked as d
 
 ## Expectation
 
-For each of the image resources referenced by the [image sources][] of the test target, any of the following is true:
+For each of the image resources referenced by the [image sources][] of the test target, at least one of the following is true:
 
 - the image resource does not contain text expressing anything in a [human language][]; or
-- the text in the image resource is not the most significant visual content; or
-- it is [essential][] that the text in the image resource is rendered with that specific presentation.
+- the image resource contains text and the text is not a significant part of the visual content in the image; or
+- the image resource contains text and it is [essential][] that the text is rendered with that specific presentation.
 
 ## Assumptions
 
-_There are currently no assumptions_
+- The text in the image resource is visible and not hidden by other resources rendered on the web page. If the text is not visible, even though parts of the image are, this rule may fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might by satisfied.
+- The page does not provide a mechanism to change the rendered text in the image resource. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might by satisfied.
+- The specific presentation of the text rendered in the image resource can be achieved through formatted text. Otherwise, the rule might fail while [SC 1.4.5 Images of Text][sc1.4.5] and [SC 1.4.9 Images of Text (No Exception)][sc1.4.9] might by satisfied.
```

Comment:
Wouldn't the photo of an old letter (used in examples) fail that assumption?
This looks like it is going to often overlap with the "essential" expectation. As in, if specific presentation is essential but can be done by CSS, it should; if specific presentation is essential but cannot be done by CSS, then OK, use an image.

Reply:
This is really context dependant. If you argue the image is just showing cursive handwriting, then yes, it could be done with CSS. If you argue the image is showing human cursive handwriting, where the same letter is never written exactly in the same way, and the lines are not really straight, and so on, then no, it cannot be done using (text formatted) CSS.

It would be up for the evaluator to judge if the assumption is met or not and decide on whether to take into account the results of the rule. If we are not happy with that for this rule/assumption, then I see only one way forward: make this a 1.4.9 only rule, where this assumption would not be needed anymore (neither the other assumptions).

Reply:
I agree with @Jym77 that this assumption can be tweaked a little. Maybe something like this:

"For images that present significant text, the specific presentation is either essential or can be presented as formatted text (eg. through CSS)."